### DISCUSSION TITLE
New Object Information: Angular distance to Sun

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#1031] [Q&A] [answered]
In the Configuration->Object information selection should be very useful to add an Angular distance to Sun or some other sort of background illumination calculation, so observation planning can be facilitated.

**Comment by alex-w** [comment]
Are you tried use AstroCalc/Ephemeris tool? Or maybe are you check elongation value?

**Comment by diegorodriguezperegrin** [comment]
I have tried AstroCalc/PC tool that shows a wonderful linear/angular graph that may help in planning too. Angular distance to Sun should anyway be a "nice to have" feature in Object information.

**Comment by diegorodriguezperegrin** [comment]
I also triedAstroCalc/Transits, that includes elongations for Sun and Moon. Still, different from AstroCalc/Ephemeris, it's something separated from the visual representation of the sky.

**Comment by gzotti** [comment, marked as answer]
Angular distance from the sun is called Elongation. It has been in the info for many years.

**Comment by diegorodriguezperegrin** [comment]
Yes, but I think that moving from main simulation window to a separate tool to get a graph sort of breaks the experience. Anyway, visibility is not all about elongation. Some sort of calculation to assess object visibility not only based on Solar angular separation but also on horizon and Moon may improve Stellarium useability. Asking too much?

**Comment by gzotti** [comment]
You asked for angular distance from the sun in the object info. This is called elongation, and is already there. No separate tool or panel required.
For visibility estimates there is the visibility plugin. This can be improved or rewritten by somebody, if somebody can find the time for it. Parts of it has been superseded by standard information strings.

**Comment by diegorodriguezperegrin** [comment]
I have seen Elongation is already available as Additional information, so this is what I needed. Thank you.
